.github/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM coqorg/coq:8.17.1-ocaml-4.14.1-flambda
```

Comment:
This docker image is outdated anyway. In fact, they, for some reason, do not support OCaml 4.14 for Coq 8.17 anymore.